controller/gui/template/network.mustache

```diff
@@ -34,20 +39,27 @@
                 </span>
             </summary>
 
-            <pre class="d-Preformatted">{{properties}}</pre>
+            <div class="d-Network__Properties">
+                <pre class="d-Preformatted">{{properties}}</pre>
+
+                {{^connected}}
+                    <form action="/network/{{id}}/connect" method="post" class="d-Network__Form">
+                        <label for="passphrase" class="d-Network__Label">Passphrase</label>
+                        <input type="text" class="d-Input d-Network__Input" id="passphrase" name="passphrase">
+
+                        <label for="proxy" class="d-Network__Label">Proxy</label>
+                        <input type="text" class="d-Input d-Network__Input" id="proxy" value="{{proxy}}" name="proxy" placeholder="192.168.1.10:8500">
```

Comment:
Maybe this section could be put into a

```html
<details>
<summary>Advanced Settings</summary>
...
</details>

Reply:
Tried with https://github.com/dividat/playos/pull/44/commits/ac6e4f1788c69cac311ac1c3f0fc5b006536cdfb.